--- 2003-11-08  2003-09-30 02:25:33.000000000 +0100
+++ 2004-01 2003-11-22 18:01:35.000000000 +0000
@@ -277,6 +277,7 @@
        
                       <div id="p">
            <ul>
+               <li>Trabalha desde 2003 na empresa UniOne como Analista Informatico de Sistemas</li>
                <li>Projectou uma base de dados para a Ourivesaria Animar Jóias</li>
            </ul>
            </div>
@@ -402,8 +403,8 @@
 </html>
 
 <!--
-     FILE ARCHIVED ON 07:11:19 Nov 08, 2003 AND RETRIEVED FROM THE
-     INTERNET ARCHIVE ON 16:52:35 Nov 18, 2023.
+     FILE ARCHIVED ON 19:08:38 Jan 11, 2004 AND RETRIEVED FROM THE
+     INTERNET ARCHIVE ON 16:51:58 Nov 18, 2023.
      JAVASCRIPT APPENDED BY WAYBACK MACHINE, COPYRIGHT INTERNET ARCHIVE.
 
      ALL OTHER CONTENT MAY ALSO BE PROTECTED BY COPYRIGHT (17 U.S.C.

